GroupDocs.Editor.MVC/Products/Editor/Controllers/EditorApiController.cs

```diff
@@ -354,114 +392,179 @@ private dynamic GetSaveFormat(string saveFilePath)
                 default:
                     format = WordProcessingFormats.Docx;
                     break;
-
             }
+
             return format;
         }
 
-        private dynamic GetSaveOptions(string saveFilePath)
+        private ILoadOptions GetLoadOptions(string guid)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GetLoadOptions' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5172828)